lambda/shared/person.go

```diff
@@ -0,0 +1,40 @@
+package shared
+
+type Address struct {
+	Line1    string `json:"line1" dynamodbav:""`
+	Line2    string `json:"line2" dynamodbav:""`
+	Line3    string `json:"line3" dynamodbav:""`
+	Town     string `json:"town" dynamodbav:""`
+	Postcode string `json:"postcode" dynamodbav:""`
+	Country  string `json:"country" dynamodbav:""`
+}
+
+type Person struct {
+	FirstNames  string  `json:"firstNames" dynamodbav:""`
+	Surname     string  `json:"surname" dynamodbav:""`
+	DateOfBirth Date    `json:"dateOfBirth" dynamodbav:""`
+	Email       string  `json:"email" dynamodbav:""`
+	Address     Address `json:"address" dynamodbav:""`
+}
+
+type Donor struct {
+	Person
+	OtherNames string `json:"otherNames" dynamodbav:""`
```

Comment:
Just curious why we have an other names field? I thought the decision was to just stick with firstnames & last name?

Reply:
I believe this is for "other names known by", like if you're called Elizabeth but everyone calls you Betty

Reply:
Worth making the field more precisely named, to avoid that confusion next time?